README.md

```diff
@@ -1,90 +1,141 @@
-* [PK Analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/pk-analysis.html)
-* [Sensitivity analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/sensitivity-analysis.html)
-* [Table parameters](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/table-parameters.html)
-* [Dimensions and Units](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/unit-conversion.html)
-* [Working with data sets and import from excel](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html)
-* [Working with `DataCombined` class](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined.html)
-* [Visualizations with `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
-
-# Installation
-
-The **ospsuite-R** package is compatible with version 3.6.x **AND** version 4.x.x of R. One of its dependency, **rClr** needs to be installed specifically for the targeted R version. Please follow the installation instructions below.
-
-**ospsuite** requires following packages to be installed:
-
-From CRAN:
-
-- [dplyr](https://cran.r-project.org/web/packages/dplyr/index.html)
-- [purrr](https://cran.r-project.org/web/packages/purrr/index.html)
-- [R6](https://cran.r-project.org/web/packages/R6/index.html)
-- [readr](https://cran.r-project.org/web/packages/readr/index.html)
-- [stringr](https://cran.r-project.org/web/packages/stringr/index.html)
-- [tidyr](https://cran.r-project.org/web/packages/tidyr/index.html)
-- [ggplot2](https://cran.r-project.org/web/packages/ggplot2/index.html)
-- [rlang](https://cran.r-project.org/web/packages/rlang/index.html)
-- [jsonlite](https://cran.r-project.org/web/packages/jsonlite/index.html)
-- [patchwork](https://cran.r-project.org/web/packages/patchwork/index.html)
-- [cowplot](https://cran.r-project.org/web/packages/cowplot/index.html)
-- [scales](https://cran.r-project.org/web/packages/scales/index.html)
-- [data.table](https://cran.r-project.org/web/packages/data.table/index.html)
-
-
-Must be downloaded manually:
-
-- rClr
-  - [For R 4.x.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.2/rClr_0.9.2.zip)
-  - [For R 3.6.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip)
-
-- [ospsuite.utils](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases/latest)
-
-- [tlf](https://github.com/Open-Systems-Pharmacology/TLF-Library/releases/latest)
-
+The **ospsuite-R** package provides the functionality of loading,
+manipulating, and simulating the simulations created in the Open Systems
+Pharmacology Software tools PK-Sim and MoBi.
+
+-   [Documentation](#documentation)
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [Known issues](#known-issues)
+-   [Code of conduct](#code-of-conduct)
+-   [Contribution](#contribution)
+-   [Licence](#licence)
+
+# Documentation {#documentation}
```

Comment:
@Felixmil Can you explain, what the anchor in curly brackets (`{#documentation}`) does? 
Is it used by pkgdown to create html anchor? Or is it already used by the GH-markdown?
And does it _replace_ the default markdown anchor (which in this concrete example would be `#documentation` as well) or adds an _additional_ anchor?

Reply:
On GitHub it does not look good:
![grafik](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/assets/25061876/efb100b0-1f01-433b-98bb-12ffe88d4e89)

In the reporting engine we use html id tags, which can be referenced both in markdown and html and are hidden when markdown is shown on GitHub. 
E.g.

`# Documentation <a id="documentation"></a>`

Reply:
That was me, will remove.